terraform/eval_diff.go

```diff
@@ -238,6 +238,38 @@ func (n *EvalDiffTainted) Eval(ctx EvalContext) (interface{}, error) {
 	return nil, nil
 }
 
+// EvalFilterDiff is an EvalNode implementation that filters the diff
+// according to some filter.
```

Comment:
Right, but I expect to expand its feature-set in the future, for now I just did what we needed.